tests/qos/qos_sai_base.py

```diff
@@ -732,7 +732,7 @@ def dutConfig(
         topo = tbinfo["topo"]["name"]
 
         # LAG ports in T1 TOPO need to be removed in Mellanox devices
-        if topo in self.SUPPORTED_T0_TOPOS or isMellanoxDevice(src_dut):
+        if topo in self.SUPPORTED_T0_TOPOS or (topo in self.SUPPORTED_PTF_TOPOS and isMellanoxDevice(src_dut)):
```

Comment:
In the past, t0-120 topo was correctly processed because it is mellanox device.
Now, since t0-120 is not in ptf_topo, need to add t0-120 topo to SUPPORTED_T0_TOPOS.
please help to review PR #10200